internal/provider/vsphere/installConfig.go

```diff
@@ -5,31 +5,17 @@ import (
 
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/installcfg"
-	"github.com/openshift/assisted-service/models"
 )
 
-func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform, clusterPlatform *models.VspherePlatform) {
-	if clusterPlatform != nil && clusterPlatform.VCenter != nil {
-		platform.VCenter = *clusterPlatform.VCenter
-		platform.Username = *clusterPlatform.Username
-		platform.Password = *clusterPlatform.Password
-		platform.Datacenter = *clusterPlatform.Datacenter
-		platform.DefaultDatastore = *clusterPlatform.DefaultDatastore
-		platform.Network = *clusterPlatform.Network
-		platform.Cluster = *clusterPlatform.Cluster
-		if clusterPlatform.Folder != nil {
-			platform.Folder = *clusterPlatform.Folder
-		}
-	} else {
-		// This is to support adding parameters in day2
-		platform.Cluster = PhCluster
-		platform.VCenter = PhVcenter
-		platform.Network = PhNetwork
-		platform.DefaultDatastore = PhDefaultDatastore
-		platform.Username = PhUsername
-		platform.Password = PhPassword
-		platform.Datacenter = PhDatacenter
-	}
+func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform) {
+	// This is to support adding parameters in day2
```

Comment:
```suggestion
	// Add placeholders to make it easier to replace in day2
```

Reply:
Changed, but it's the original code, I just removed the if section